src/presenter/LoginPresenter.js

```diff
@@ -0,0 +1,22 @@
+import userModel from "../model/UserModel";
+
+class LoginPresenter {
+    onChange(property, value) {
+        userModel.changeNewUserProperty(property, value);
+    }
+    onLogin(username, password) {
+        console.log("LoginPresenter onLogin");
+        var user = userModel.state.users.find(user =>
```

Comment:
This find method should be inside the user model (as it is a sort of business operation),